src/js/modules/supervisors/HandleError.ts

```diff
@@ -48,34 +59,61 @@ export const validateDisableResponseCode = (
 };
 
 export const validateEnableResponseCode = (
-  responses: EnableCoprosReply
+  responses: EnableCoprosReply,
+  coprocessors: Coprocessor[]
 ): Error[][] => {
-  return responses.inputs.map((response) => {
+  if (responses.inputs.length !== coprocessors.length) {
+    throw new Error(
+      "error: inconsistent response for enable coprocessors, " +
+        "the enable coprocessor response for " +
+        coprocessors.join(", ") +
+        "doesn't have the same number item results, expected: " +
```

Comment:
There should be a leading space in this string.

Reply:
Done